backend/activities/api/urls.py

```diff
@@ -1,4 +1,7 @@
 from django.urls import path
+from .views import GetActivitiesAPI, GetAllActivitiesAPI
 
 urlpatterns = [
+    path('', GetAllActivitiesAPI.as_view()),
+    path('<int:pk>', GetActivitiesAPI.as_view()),
```

Comment:
its kinda odd if the path is empty. In your main urls.py, pls change it to `/activity` without trailing slash at the front most from this '/activity/' for example. Pls also change necessary changes. 
```suggestion
    path('/', GetAllActivitiesAPI.as_view()),
    path('/<int:pk>', GetActivitiesAPI.as_view()),
```